priorityclass/priorityclass.go

```diff
@@ -0,0 +1,59 @@
+package priorityclass
+
+import (
+	"context"
```

Comment:
Please sort your import with `make lint-fix`.

Reply:
I've issued the command on the root of the project

Reply:
But it doesn't seem like it's doing much

Reply:
It's ok. All imports are sorted correctly.